tika/src/test/java/fr/pilato/elasticsearch/crawler/fs/tika/TikaDocParserTest.java

```diff
@@ -667,6 +667,38 @@ public void testOcr() throws IOException, TikaConfigException {
         }
     }
 
+    @Test
+    public void testCustomTikaConfig() throws IOException, URISyntaxException {
+        InputStream tikaConfigIS = getClass().getResourceAsStream("/documents/tikaConfig.xml");
```

Comment:
See the previous comment about the location of this file.

Reply:
For the unit test I am still copying to a temporary file, rather than accessing the file directly (although I have used the tmp directory you've recommended). I could not find a way to access the resource in the jar without copying it to a new location. Please let me know if I've missed something.